frontend/src/components/ui/LoadMoreButton.tsx

```diff
@@ -6,19 +6,21 @@ export default function LoadMoreButton({
   isLoading,
   setSize,
   size,
+  entity,
```

Comment:
```suggestion
  type,
```
`type` makes more sense imo